hr_employee_id/models/res_company.py

```diff
@@ -25,5 +25,5 @@ class ResCompany(models.Model):
     employee_id_sequence = fields.Many2one(
         comodel_name='ir.sequence',
         string='Sequence',
-        help="Pattern to be used for used for ID Generation",
+        help="Pattern to be used for used for ID Generation"
```

Comment:
@astirpe Hi Andrea, can you please fix the help message with better description: Pattern to be used for employee ID Generation".